lib/models/patient.dart

```diff
@@ -104,7 +104,7 @@ class Patient extends AbstractDatabaseObject {
     // A non-negative number for determining the length of the password.
     const int length = 12;
 
-    Random randomString = Random();
+    Random randomString = Random.secure();
```

Comment:
You shouldn't mix up branches like that, at least if they don't rely on each other.

Reply:
> You shouldn't mix up branches like that, at least if they don't rely on each other.

"Resolved in commit [c89841f](https://github.com/healthcare-it-solutions/picos/pull/128/commits/c89841f909a7f54d1302d54011599995ca2f539d)"